rpc_jobs/rpc_aio.yml

```diff
@@ -180,6 +183,20 @@
       # Ocata onwards.
       - series: master
         image: trusty
+      # Ironic builds are not supported for series before newton
```

Comment:
Something we've been exploring recently is adding new JJB projects that used the same template as previous projects, this can help reduce the complexity of projects and the number of excludes that are required. 

See https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/irr_role_test.yml#L56-L69 for an example where ceph jobs were added as a separate project rather than new axis/item to the existing project.

Reply:
Done!